bigbluebutton-html5/imports/ui/components/app/service.js

```diff
@@ -91,6 +91,11 @@ function observeBreakoutEnd() {
   });
 }
 
+function meetingIsBreakout() {
+  const breakouts = Breakouts.find().fetch();
+  return (breakouts && breakouts.filter(b => b.breakoutMeetingId === Auth.meetingID).length != 0);
```

Comment:
Could use the `some` array method instead of filter.
`breakouts.some(b => b.breakoutMeetingId === Auth.meetingID)`